common/app/views/fragments/standfirst.scala.html

```diff
@@ -38,4 +38,15 @@
             )
         }
     }
+    @if(item.tags.isAudio) {
+        <p class=".content__standfirst">
+            <span class="bullet"></span>
+            <a href="https://www.theguardian.com/media/2017/oct/07/how-to-listen-to-podcasts-everything-you-need-to-know?CMP=podcast-help"
```

Comment:
Yes definitely! It would look like this:

`@LinkTo("/media/2017/oct/07/how-to-listen-to-podcasts-everything-you-need-to-know?CMP=podcast-help")"`

This ensures that on code you link to the code version vs prod where you link to the prod version etc.

At the top of the file you will also need to import it

`@import common.LinkTo`